### ISSUE TITLE
0-bootstrap needs a setup.sh script - for multitenant or developer reruns - as well as customer throughput/repeatability

### ISSUE THREAD

**Issue by obriensystems** [terraform-google-modules/terraform-example-foundation#1144]
### TL;DR

see all issues with 0-bootstrap

- https://github.com/terraform-google-modules/terraform-example-foundation/issues/1135
- https://github.com/terraform-google-modules/terraform-example-foundation/issues/1136
- https://github.com/terraform-google-modules/terraform-example-foundation/issues/1137
- https://github.com/terraform-google-modules/terraform-example-foundation/issues/1138
- https://github.com/terraform-google-modules/terraform-example-foundation/issues/1139
- https://github.com/terraform-google-modules/terraform-example-foundation/issues/1140
- https://github.com/terraform-google-modules/terraform-example-foundation/issues/1141
- https://github.com/terraform-google-modules/terraform-example-foundation/issues/1142
- https://github.com/terraform-google-modules/terraform-example-foundation/issues/1143

### Terraform Configuration

```hcl
shell
```

### Terraform Version

```sh
1.7.4
```

**Comment by fmichaelobrien** [comment]
0-bootstrap is up with the changes from #1133 to #1143
<img width="1397" alt="Screenshot 2024-03-07 at 10 12 58" src="https://github.com/GoogleCloudPlatform/pbmm-on-gcp-onboarding/assets/24765473/c92ecd3d-f72d-4106-9fd4-801953156d13">

**Comment by fmichaelobrien** [comment]
stale bot timer restart - https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/.github/workflows/stale.yml#L21
